jerry-core/parser/regexp/re-compiler.h

```diff
@@ -57,8 +57,14 @@
 #define RE_OP_CHAR_CLASS                                    26
 #define RE_OP_INV_CHAR_CLASS                                27
```

Comment:
@ruben-ayrapetyan, it looks like other parts of the engine are using enums instead of defines. Should I change definitions of RegExp opcodes and tokens to enum?